department-of-veterans-affairs/caseflow
# Updated AppealInformation component

Resolves [CASEFLOW-2073](https://vajira.max.gov/browse/CASEFLOW-2073)

**PR Stack**

Part 1
[Part 2](https://github.com/department-of-veterans-affairs/caseflow/pull/16662)

### Description

Updates the schedule veteran form with the new Designs (see [Video/Central hearing version](https://www.figma.com/file/8mwL6nupiylRtspfs8Qvru/Emails-for-all-hearings-(2021-PI8)?node-id=156%3A508) and [Virtual hearing version](https://www.figma.com/file/8mwL6nupiylRtspfs8Qvru/Emails-for-all-hearings-(2021-PI8)?node-id=161%3A719))

### Acceptance Criteria
- [x] The text " (for these notifications only)" is added to the end of the appellant and POA email labels
e.g., "POA / Representative Email (for these notifications only)"
- [x] The help text under the Veteran/appellant and POA email inputs are updated to read "This email address will be used to send notifications for this hearing only."
- [x] Replace "Appeal Information" heading text with appellant/Veteran name
- [x] Display appellant/Veteran mailing address directly under appellant name heading
- [x] Display "Relation to Veteran" subheading and value between appellant address and Issues
- [x] Replace static "Power of Attorney" subheading with dynamic subheading that uses the type of representative as the text ("Attorney", "Agent", "Service Organization", with "Power of Attorney" as default for "Other")
- [x] Display representative name and address under that dynamic representative subheading
- [x] The helper label under the "Schedule Veteran for a Hearing" heading no longer appears when scheduling a virtual hearing
- [x] The Veteran/Appellant Name, Relation to Veteran, and Appellant/Veteran address are no longer shown in the appellant section of Email Notifications
- [x] The representative type, name, and address are no longer shown in the representative section of Email Notifications
- [x] The Veteran/Appellant mailing address no longer appears above the Regional Office dropdown when scheduling a Video hearing

### Testing Plan

1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearing schedule: http://localhost:3000/hearings/schedule
3. Click the “Schedule Veteran” button at the top left
4. Select the St. Petersburg, FL RO from the dropdown
5. Click the “AMA Veterans Waiting” tab and click the name of one of the Veterans
6. On the Case Details page, click the “Schedule Veteran” option from the task list
7. Ensure that all of the form fields match the changes in the Figma diagrams above for both Virtual and Video/Central

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1716" alt="BeforeVideoScheduleVeteran" src="https://user-images.githubusercontent.com/61989526/130272786-1c7ce966-6289-4d43-8e44-2da063aa79e2.png">|<img width="1674" alt="AfterVideoScheduleVeteran" src="https://user-images.githubusercontent.com/61989526/130272811-9c9a4c96-ac98-4e43-b3da-ef815e11215d.png">
<img width="1732" alt="BeforeVirtualScheduleVeteran" src="https://user-images.githubusercontent.com/61989526/130272831-5451c3a4-c296-45f8-8ad4-daba9f381b18.png">|<img width="1705" alt="AfterVirtualScheduleVeteran" src="https://user-images.githubusercontent.com/61989526/130272839-d5f8838a-e29b-4870-ac96-7cf02ed59480.png">
